### PR TITLE
Change IAM role for spot fleet instances

### PR DESCRIPTION
**Pull request by averkamp-rmn**
Add ability to make selenium hosted zone public.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,15 +17,16 @@ STACK_PARAMETERS=--parameters ParameterKey=VpcId,ParameterValue="$(ECS_SELENIUM_
...
 
 
 create-stack:
-	aws cloudformation create-stack \ 
+	aws cloudformation create-stack \
 		--region $(AWS_REGION) \
 		--stack-name $(ECS_SELENIUM_STACK_NAME)  --capabilities CAPABILITY_NAMED_IAM \
 		--template-body file://./cloudformation/ecs-selenium.cfn.yml \
-		$(STACK_PARAMETERS)
+		$(STACK_PARAMETERS)
```
Comment (andybug) [inline Makefile]: Whitespace

Comment (averkamp-rmn) [reply Makefile]: Removed whitespace

**Review by andybug — APPROVED** [review]

**Comment by achuinard** [comment]
This PR broke deployment, I get an error regarding the SeleniumHostedZone block in the Cloud Formation YAML, the error in AWS is that VPCs must be a list. I had to manually modify my Cloud Formation to look like what it did before in this PR.

**Comment by achuinard** [comment]
```
VPCs: 
    - !If [ShouldCreatePrivateHostedZone, [VPCId: !Ref VpcId, VPCRegion: "AWS::Region"], !Ref 'AWS::NoValue']
```

This is the new block that is broken.

**Comment by achuinard** [comment]
@andybug @averkamp-rmn

**Comment by averkamp-rmn** [comment]
Thank you for reporting this, I have tested a fix and posted a pr to fix it here:

https://github.com/RetailMeNotSandbox/ecs-selenium/pull/34

**Pull request merged**